go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	github.com/google/uuid v1.1.2
 	github.com/gophercloud/gophercloud v0.7.0
 	github.com/gophercloud/utils v0.0.0-20200204043447-9864b6f1f12f
+	github.com/onsi/ginkgo v1.16.4
```

Comment:
Hm, ginkgo@v1 was recently replaced by ginkgo@v2 -> you need to adapt your tests to use ginkgo@v2 imports. We don't want to introduce again ginkgo@v1.

```suggestion
```